index.html

```diff
@@ -14,67 +14,73 @@
 
 <body>
     <header>
-        <h1>Jiachun</h1>
+        <h1 aria-label="Jiachun" >Jiachun</h1>
         <nav>
-            <img class="header-mobile" src="./img/bar-icon.png" />
+            <img class="header-mobile" src="./img/bar-icon.png" alt="bar-icon" />
             <ul>
                 <li>Profolio</li>
```

Comment:
https://github.com/mpa-mxiang/profolio2/blob/7e82c2eb2f8d52c8e4e86a74b6e37b9f6d981ac7/index.html#L12

- [ ] Your page title should be spelled correctly and I suggest that it include your name, for example, "John's Portfolio Website"